tracing/tracer.go

```diff
@@ -0,0 +1,94 @@
+package tracing
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/opentracing/opentracing-go"
+	zipkinot "github.com/openzipkin-contrib/zipkin-go-opentracing"
+	zipkin "github.com/openzipkin/zipkin-go"
+	"github.com/openzipkin/zipkin-go/reporter"
+	zipkinhttp "github.com/openzipkin/zipkin-go/reporter/http"
+)
+
+var zipkinReporter reporter.Reporter
+
+// Tracer is exported to support start tracible server etc
+var Tracer = opentracing.GlobalTracer()
+
+// InitTracer configure the global tracer
+func InitTracer(serviceName, addr string, sampleRate float64) {
+	if addr == "" {
+		// TODO: log info level
+		return
+	}
+
+	zipkinReporter := zipkinhttp.NewReporter(addr)
+
+	endpoint, err := zipkin.NewEndpoint(serviceName, "0.0.0.0:0")
+	if err != nil {
+		log.Fatalf("unable to create local endpoint: %+v\n", err)
```

Comment:
Let's not use the stdlib log in library code. Instead, pass in a log [`Wrapper`](https://godoc.org/github.com/reddit/baseplate.go/log#Wrapper), use that to do logging (example: https://github.com/reddit/baseplate.go/blob/903bbee5dcd49766b720b2bbd585dfa88f9f7195/filewatcher/filewatcher.go#L71-L73), and return an error here.

Or we can just return the error back to the caller and let the caller to decide what to do (logging and/or panic).

Same as the other log usages.